package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-create-ui",
-  "version": "5.2.8",
+  "version": "5.2.9",
```

Comment:
As with 16552, this is a fairly significant change, so update the minor version (5.3.0).